src/main/java/frc/robot/RobotContainer.java

```diff
@@ -32,18 +37,27 @@
   public static DriveSubsystem mDriveSubsystem = new DriveSubsystem();
   public static DriveCommand mDriveCommand = new DriveCommand();
   private static final Logger LOGGER = Logger.getLogger(Robot.class.getName());
-  public static Path path = new Path();
-  public static ArrayList leftArray;
-  public static ArrayList rightArray;
+  public static Path path1 = new Path("/paths/StraightTen.");
+  public static Path path2 = new Path("/paths/Turn.");
+  public static ArrayList leftArray1;
+  public static ArrayList rightArray1;
+  public static ArrayList leftArray2;
+  public static ArrayList rightArray2;
+  public static NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+  public static NetworkTableEntry tx = table.getEntry("tx");
+  public static NetworkTableEntry ty = table.getEntry("ty");
+  public static NetworkTableEntry ta = table.getEntry("ta");
```

Comment:
Your ArrayLists should have a <String> container type.
Also seems like we could rewrite this to make adding new paths easier. 
Also also I think most of these variables shouldn't be public